reactive_kernel/kernel.py

```diff
@@ -1,6 +1,8 @@
 from ipykernel.kernelbase import Kernel
 import sys
-from .codeObject import CodeObject
+from .code_object import CodeObject
```

Comment:
I would `import logging` and then after all of the imports put:

`KERNEL_LOGGER = logging.getLogger(__name__)`